### DISCUSSION TITLE
Wikijs 2.x and 16.x LTS end of life on 11 Sep 2023

### DISCUSSION THREAD

**Discussion by tgurr** [requarks/wiki#6642] [Help / Questions] [answered]
Node.js 16.x LTS will reach its end of life on 11 Sep 2023 creating a possible security issue in the nearby future especially running in corporate environments where there may be policies in regards to running end of life software and/or distributions having "packaged" (https://repology.org/project/wiki-js/versions) wikijs which also (may) have policies to remove unsupported end of life packages. Since Wikijs 2.x currently doesn't support >= 18.0.0 (https://github.com/requarks/wiki/commit/e1d282ad1102fb298528238d3c6f18f073dbeb7d), will we get a timely update to be able to run Wikijs with a still supported Node.js version?

**Comment by NGPixel** [comment, marked as answer]
Yes, an update is already in the works and should be available soon.

**Reply by NGPixel** [reply]
v2.5.300 is now available with support for both Node 18 and 20.

**Reply by tgurr** [reply]
Awesome, thank you very very much!

**Reply by prozac64** [reply]
I can not get Wiki.js to start with Node.js v18/v20
